src/sql/src/plan/statement/ddl.rs

```diff
@@ -1833,13 +1833,16 @@ pub fn plan_view(
     let query::PlannedQuery {
         mut expr,
         mut desc,
-        finishing,
+        // We get back a trivial finishing, because `plan_root_query` applies the given finishing.
+        // Note: Earlier, we were thinking to maybe persist the finishing information with the view
+        // here to help with materialize#724. However, in the meantime, there might be a better
+        // approach to solve materialize#724:
+        // https://github.com/MaterializeInc/materialize/issues/724#issuecomment-1688293709
+        finishing: _,
```

Comment:
Add a `assert!(finishing.is_trivial())` so we don't mess this up.

Reply:
Good idea, added.